robyn/__init__.py

```diff
@@ -159,24 +159,26 @@ def startup_handler(self, handler: Callable) -> None:
     def shutdown_handler(self, handler: Callable) -> None:
         self._add_event_handler(Events.SHUTDOWN, handler)
 
-    def start(self, url: str = "127.0.0.1", port: int = 8080):
+    def start(
+        self, host: str = "127.0.0.1", port: int = 8080, *, url: str = "127.0.0.1"
+    ):
         """
         Starts the server
 
         :param port int: represents the port number at which the server is listening
         """
 
-        url = os.getenv("ROBYN_URL", url)
+        host = os.getenv("ROBYN_HOST", host) or os.getenv("ROBYN_URL", url)
```

Comment:
I think an `or` condition here is wrong. If someone passes `url="0.0.0.0"` it will still be `127.0.0.1` due to `host`'s default value.